.travis.yml

```diff
@@ -36,7 +42,8 @@ extended-test-suite: &extended-test-suite
 matrix:
   include:
     # Main test suite
-    - python: "2.7"
+    - stage: "Test"
```

Comment:
As it also indicated in the example code here: https://docs.travis-ci.com/user/build-stages/#how-to-define-build-stages, I think it would be useful to remind the reader that the `Test` stage will be applied to each next matrix object until another stage is explicitly set.